Minor fixes to spelling and grammar (#98)
se-edu/se-book: gitAndGithub/commit/sourcetree_3.md

> Click the `Commit` button, enter a commit message e.g. `add fruits.txt` in to the text box, and click `Commit`.

Reviewer: ```suggestion
Click the `Commit` button, enter a commit message e.g. `add fruits.txt` into the text box, and click `Commit`.
```
